packages/story-editor/src/components/floatingMenu/elements/karma/textAlign.karma.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../../karma';
+import { useStory } from '../../../../app/story';
+
+describe('Design Menu: Text alignment', () => {
+  let fixture;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    fixture.setFlags({ floatingMenu: true });
+    try {
+      await fixture.render();
+    } catch {
+      // ignore
```

Comment:
Ah, it's because otherwise it doesn't work on my local setup (because of some font loading error thrown). I have to remember to remove these before pushing. Fixed in a145fc96d1.